src/Universum/Container/Class.hs

```diff
@@ -722,12 +671,6 @@ type family DisallowInstance (z :: Symbol) :: ErrorMessage where
         :$$: Text ""
 #endif
 
-#define DISALLOW_TO_LIST_8(t, z) \
-    instance TypeError (DisallowInstance z) => \
-      ToList (t) where { \
-        toList = undefined; \
-        null = undefined; } \
-
 #define DISALLOW_CONTAINER_8(t, z) \
     instance TypeError (DisallowInstance z) => \
       Container (t) where { \
```

Comment:
Probably, this can be simplified as there are default implementations for these functions

Reply:
I don't mind. We can make this scary CPP pragma much less scarier.